src/components/Taxonomy/Taxonomy.tsx

```diff
@@ -183,15 +203,14 @@ const Item = ({ item, isFiltering }: { item: TaxonomyItem, isFiltering?: boolean
             disabled={disabled}
             checked={checked}
             ref={setIndeterminate}
-            onChange={e => setSelected(item.path, e.currentTarget.checked)}
+            onChange={e => setSelected(path, e.currentTarget.checked)}
           />
-          {item.label}
+          {name}
         </label>
         {!isFiltering && (
           <div className={styles.taxonomy__extra}>
-            {/* @todo should count all the nested children */}
             <span className={styles.taxonomy__extra_count}>
-              {item.children?.length}
+              {childCount}
```

Comment:
cool! that's the count of all the descendants?

Reply:
YUS!!